nototools/generate_website_2_data.py

```diff
@@ -554,7 +554,7 @@ def get_css_generic_family(family):
 
 CSS_WEIGHT_MAPPING = {
     'Thin': 250,
```

Comment:
What does this mapping do? Shouldn't Thin be mapped to 100 (even though inside the font, OS/2.fsWeight is set to 250) for CSS?